lstmcpipe/config/pipeline_config.py

```diff
@@ -196,7 +191,7 @@ def complete_lstmcpipe_config(loaded_config):
         "PathConfigAllSky": "prod_all_sky",
     }
 
-    suffix_id = "_{}_{}".format(all_mc_prods[prod_type], sufix_prod_id)
+    suffix_id = "_{}".format(suffix_prod_id)
```

Comment:
Why taking out the {`prod5`, `prod5_trans_80`, `prod_all_sky`} from the prod_id ? 
In this case remove the `all_mc_prods` dict too, as it is no longer used.

Reply:
I really don't  understand the need for this, other that logging.
But logs contain the config, which has this information.

Reply:
Filenaming of individual files and dirs will be shorter, it will be less self explicative. Nobody will go to the logs (that are indeed where exactly ?) to check if a prod is prod5, prod5_trans_80 if it is not written on the dir, imo

Reply:
![Screenshot 2022-04-10 at 00 17 06](https://user-images.githubusercontent.com/40355845/162593587-5b1f5048-38eb-49d4-9be5-34a851207baf.png)

Reply:
we need to be consistent with naming of dirs (we create to many in too many different places). To be discussed

Reply:
Ok for the directory naming that will help people navigating the dir tree.
I have added `prod_type` in the config in #217 . It can be used directly for logging and naming purposes.